### DISCUSSION TITLE
Error with Pymol

### DISCUSSION THREAD

**Discussion by pennysr** [mdtraj/mdtraj#1868] [General]
I am not really the end user, I am the IT guy, I have a reasonable grasp of Linux in general but not when it gets too in depth.

One of our people is using Pymol and getting the error:: 

NameError: free variable 'mdtraj' referenced before assignment in enclosing scope

I am not really sure if this is a config issue, a bug or something they are entering incorrectly. Any ideas?

**Comment by mattwthompson** [comment]
No, I don't have any idea what could be causing past the contents of that error message

**Reply by ctk3b** [reply]
Given the error message, this is likely a bug in pymol and I recommend posing your question on their repo https://github.com/schrodinger/pymol-open-source

If it is somehow an mdtraj issue please report back here!
